vnext/Microsoft.ReactNative.Cxx/ReactPropertyBag.h

```diff
@@ -98,35 +92,34 @@ struct ReactPropertyNamespace {
     return static_cast<bool>(m_handle);
   }
 
+  static ReactPropertyNamespace Global() noexcept {
+    return ReactPropertyNamespace{ReactPropertyBagHelper::GlobalNamespace()};
+  }
+
+  hstring NamespaceName() const noexcept {
+    return m_handle ? m_handle.NamespaceName() : hstring{};
+  }
+
  private:
   IReactPropertyNamespace m_handle;
 };
 
-// Encapsulates the IReactPropertyName and the property type
-template <class T>
-struct ReactPropertyId {
-  using PropertyType = T;
-
-  ReactPropertyId(std::nullptr_t = nullptr) noexcept {}
+// ReactPropertyName encapsulates the IReactPropertyName.
+// It represents an atomic property name object that defines a LocalName
+// within the referenced Namespace.
+struct ReactPropertyName {
```

Comment:
the name Name would seem to hint that this is the leaf name i.e. the local name (whereas `Id` can be global). Curious to understand the reasoning behind the rename?

Reply:
I need to reuse it for the ReactNotificationId. Plus, it became a little bit more efficient since we have less method implementations that depend on T.

---
In reply to: [427651462](https://github.com/microsoft/react-native-windows/pull/4953#discussion_r427651462) [](ancestors = 427651462)